packages/eslint-config-base/package.json

```diff
@@ -1,10 +1,9 @@
 {
   "name": "@oacore/eslint-config-base",
-  "version": "0.5.0",
```

Comment:
You don't need to bump the version. It's done for you automatically when you release it.

See
https://github.com/oacore/configs/pull/6
https://github.com/oacore/configs/blob/master/.github/workflows/publish-packages.yml#L32

Reply:
Thank you! It looks good.

However, I am not sure this is good idea to release all packages when only one has been changed.

I know Babel uses mono-repository approach. They have [this setup](https://github.com/babel/babel/blob/master/Makefile#L198-L200). Honestly, I do not know what does it mean because I have not been working with `lerna`. Would you like to investigate? Do you have any other idea?

Reply:
AFAIK babel uses the same approach. They release package even when it wasn't changed.